modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java

```diff
@@ -557,6 +527,63 @@ protected void addEventProperties(Event event, ObjectNode propertiesNode) {
         }
     }
 
+    protected void addLocalizationProperties(UserTask userTask, ObjectNode propertiesNode) {
+        List<ExtensionElement> localizationElements = userTask.getExtensionElements().get("localization");
+        if (localizationElements != null) {
+            ObjectNode localizationNode = objectMapper.createObjectNode();
+            ObjectNode nameNode = localizationNode.putObject("name");
+            ObjectNode descriptionNode = localizationNode.putObject("description");
+            for (ExtensionElement localizationElement : localizationElements) {
+                if (FLOWABLE_EXTENSIONS_PREFIX.equals(localizationElement.getNamespacePrefix()) ||
+                        ACTIVITI_EXTENSIONS_PREFIX.equals(localizationElement.getNamespacePrefix())) {
+                    String locale = localizationElement.getAttributeValue(null, "locale");
+                    String name = localizationElement.getAttributeValue(null, "name");
+                    String documentation = null;
+                    List<ExtensionElement> documentationElements = localizationElement.getChildElements().get("documentation");
+                    if (documentationElements != null) {
+                        for (ExtensionElement documentationElement : documentationElements) {
+                            documentation = StringUtils.trimToNull(documentationElement.getElementText());
+                            break;
+                        }
+                    }
+                    if (name != null && !name.trim().isEmpty()) {
+                        nameNode.put(locale, name);
```

Comment:
If you are expecting leading/trailing spaces (the use of trim()); wouldn't you want to store the trimmed value too?

Reply:
yes, actually I simplified the if condition using StringUtils as well.

Reply:
thanks